ecommerce/api.py

```diff
@@ -472,3 +472,67 @@ def unenroll_learner_from_order(order_id):
                     )
             except ObjectDoesNotExist:
                 pass
+
+
+def check_pending_orders_for_resolution():
```

Comment:
It's fine too but since this function and the command both not only check the order status but also fix them too. Would it be a good idea to rename things e.g. `process_pending _orders` or something similar?

I'm fine if you leave it as is.

Reply:
I changed it